Isabelle/Chi_Calculus_Examples/Diamond.thy

```diff
@@ -1,7 +1,413 @@
 section \<open>Diamond-Shaped Relaying Network\<close>
 
+(* FIXME: Fill holes When bisimilarity reasoning is supported by the simplifier. *)
+
 theory Diamond
-  imports Chi_Calculus.Relaying Chi_Calculus.Proper_Weak_Transition_System Chi_Calculus.Basic_Weak_Transition_System
+  imports
+    Chi_Calculus.Relaying
+    Chi_Calculus.Proper_Weak_Transition_System
+    Chi_Calculus.Basic_Weak_Transition_System
+    Tools
```

Comment:
The theory `Tools` is apparently missing in the commit. Could you please amend the commit with the addition of this theory?

Reply:
Fixed.